<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #e9edf0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="clearBut">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 78, 116);
font: 700 16pt &quot;Times New Roman&quot;;
color: #e9edf0;
border: 3px solid #184161;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="countBut">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 78, 116);
font: 700 16pt &quot;Times New Roman&quot;;
color: #e9edf0;
border: 3px solid #184161;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m1Value">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>50</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
background-color: rgb(138, 187, 226);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x1Value">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>50</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
background-color: rgb(138, 187, 226);
border-radius: 10px;
border-color: rgb(28, 78, 116);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="n1Value">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>50</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
background-color: rgb(138, 187, 226);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelN1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelM1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x2Value">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>130</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
background-color: rgb(138, 187, 226);
border-radius: 10px;
border-color: rgb(28, 78, 116);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="n2Value">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>130</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
background-color: rgb(138, 187, 226);
border-radius: 10px;
border-color: rgb(28, 78, 116);</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelN2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelM2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m2Value">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>130</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
background-color: rgb(138, 187, 226);
border-radius: 10px;
border-color: rgb(28, 78, 116);</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIMG">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/Screenshot_1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>310</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>The result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResultValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
